src/3_5_Unic_elements.py

```diff
@@ -0,0 +1,15 @@
+"""Уникальные элементы в списке
+Дан список. Выведите те его элементы, которые
+встречаются в списке только один раз. Элементы
+нужно выводить в том порядке, в котором они
+встречаются в списке."""
+
+
+list1 = [1, 2, 2, 3, 7, 5, 6, 4, 1, 5, 6, 'f']
+
+list2 = list1.copy()
+list2.reverse()
+
+for element in list1:
+    if (len(list1) - list2.index(element) - list1.index(element)) == 1:
```

Comment:
it's task for dict. dct[a] = dct.get(a) +1 i.e. 
you solution O(n**2). p.11 review not addressed.